src/test/java/com/sonyericsson/jenkins/plugins/bfa/sod/ScanOnDemandBaseActionTest.java

```diff
@@ -66,6 +67,11 @@
     @Rule
     public JenkinsRule j = new JenkinsRule();
 
+    @Before
+    public void setup() {
+        PluginImpl.getInstance().setSodVariables(new ScanOnDemandVariables());
```

Comment:
Smells to me like something is wrong here if you have to add this?

Reply:
ScanOnDemandVariables aren't set by default in the current setup, probably needs a readResolve fix to initialise them if they aren't set

Reply:
I've pushed a fix, readResolve doesn't seem to work in the test, I added readResolve and a default initialiser to be safe